src/me/StevenLawson/TotalFreedomMod/Commands/Command_nightvision.java

```diff
@@ -0,0 +1,33 @@
+package me.StevenLawson.TotalFreedomMod.Commands;
+
+import net.md_5.bungee.api.ChatColor;
```

Comment:
Yeah, what you are using here is for a Bungee, not sure how that import even came up in all honesty because you shouldnt have a Bungee library at all.